silky-squirrels/users/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from chat.models import Room
+from PIL import Image
+from django.utils import timezone
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    image = models.ImageField(default="default.jpg", upload_to="profile_pics")
+    background = models.ImageField(upload_to="background")
+    status = models.CharField(default=":-)", max_length=100)
+    friends = models.ManyToManyField("self", blank=True)
+
+    def __str__(self):
+        return f"{self.user.username} Profile"
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
```

Comment:
nice to see that this is remembered. since you never know what you could break in the future.